apps/backend/src/mcp/tools/proxy_management.py

```
@@ -0,0 +1,698 @@
+            if not device:
+                device_query = select(
+                    ProxyConfig.device_id,
+                    func.count()
+                ).group_by(ProxyConfig.device_id)
+                
+                device_result = await session.execute(device_query)
+                device_counts = dict(device_result.fetchall())
+            
+            # SSL enabled count
+            ssl_query = select(func.count()).where(
+                ProxyConfig.parsed_config['ssl_enabled'].astext.cast(bool) == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `...:` for truth checks

Replace comparison

(E712)